R/realtime.service.R

```diff
@@ -0,0 +1,182 @@
+                       "inland waters. The public should not base evacuation decisions on this product. Citizens",
+                       "should follow the evacuation advice of local emergency management authorities.")
+    full.publications <- list(data=list(),publications=list(),resources=list())
+    keywords <- ""
+  } else {
+    tiny.text <- paste(titleMap$medium[[subType]], "during", paste0(titleParts$name, ":"), "NHC Adv.", titleParts$advNum)
+    medium.title <- paste(titleMap$medium[[subType]])
+    
+    if (subType == "pcol" | subType == "povw" | subType == "pind") {
+      description <- getPCOIDescription(subType, titleParts, doc)
+      full.title <- paste(titleMap$medium[[subType]], "during", paste0(titleParts$name, ":"), titleParts$advFull)
+      medium.text <- description$medium
+      full.text <- description$full
+    } else if (subType == "dhigh" | subType == "dlow") {
+      collectionDate <- extractCollectionDate(doc, subType)
+      attrDef <- ifelse(subType == "dhigh", 
+        "The elevation of the dune crest, or top of the foredune,",
+        "The elevation of the dune toe, or ocean-side base of the foredune,")
+      full.title <- paste(titleMap$medium[[subType]], "prior to", titleParts$name)
+      medium.text <- paste(titleMap$medium[[subType]], "(m, NAVD88) for open coast sandy beaches every 1 km alongshore.")
+      full.text <- paste("This dataset contains", tolower(titleMap$medium[[subType]]),
+                         "(m, NAVD88) for the United States coastline.", attrDef,
+                         "was extracted for open coast sandy beaches from gridded",
+                         "lidar topography every 10 m alongshore and then averaged",
+                         "in 1-km bins. Lidar surveys were collected from",
+                         paste0(collectionDate, "."))
+    } else if (subType == "mean" | subType == "extreme") {
+      surge <- getSurgeDescription(doc)
+      full.title <- paste("Modeled", tolower(titleMap$medium[[subType]]), "during",
+                          paste0(titleParts$name), titleParts$advFull)
+      medium.text <- paste("The storm-induced", ifelse(subType == "mean",
+                           "mean water levels,", 
+                           "extreme (98% exceedance) water levels,"),
+                           "at the shoreline for", paste0(titleParts$name, ": ", titleParts$advFull, "."))
+      full.text <- paste("This dataset contains modeled storm-induced", ifelse(subType == "mean",
+                             "mean water levels", 
+                             "extreme (98% exceedance) water levels"),
+                         "at the shoreline during", paste0(titleParts$name, "."),
+                         "Values were computed by summing modeled storm",
+                         paste0("surge and parameterized wave", ifelse(subType == "mean",
+                         "setup, the increase in mean water level at the shoreline due to breaking waves.", "runup.")),
```

Comment:
For mean, put space before setup, coming through mashed together like 'wavesetup'